target/bin/debug-fetchmail

```diff
@@ -1,6 +1,9 @@
 #! /bin/bash
 
-/usr/local/bin/setup-fetchmail
+# shellcheck source=../scripts/startup/setup-stack.sh
+source /usr/local/bin/setup-stack.sh
```

Comment:
`_setup_fetchmail` uses `_log` which isn't available, when just sourcing `setup-stack.sh`.

```suggestion
# shellcheck source=../scripts/helpers/log.sh
source /usr/local/bin/helpers/log.sh
# shellcheck source=../scripts/startup/setup-stack.sh
source /usr/local/bin/setup-stack.sh
```

Reply:
Nice catch! I resolved the conflicts with `master` and then included these changes as well. Done in ba8b6261a85428275435cd279ca18124d84b42d4